HW_1/task1.py

```diff
@@ -0,0 +1,7 @@
+# Задача 2: Найдите сумму цифр трехзначного числа.
+# *Пример:*
+# 123 -> 6 (1 + 2 + 3)
+# 100 -> 1 (1 + 0 + 0) |
+
+a = int (input(" Введите трехзначное число "))
+print("Сумма цифр трехзначного числа равна = ", a // 100 + ((a // 10) % 10)+ a %10)
```

Comment:
У операций деления одинаковый приоритет, то есть скобки не нужны.
```suggestion
print("Сумма цифр трехзначного числа равна = ", a // 100 + a // 10 % 10)+ a % 10)
```